### DISCUSSION TITLE
How can i tell it where my imap server is ?

### DISCUSSION THREAD

**Discussion by primski** [the-djmaze/snappymail#1118] [Q&A]
I am using Nextcloud integration.

Tries to connect to localhost but it's not there ... 

This folder also doesn't exist on my nextcloud server

```
Configuration file application.ini is found within directory structure of a special kind, like this:
/var/www/rainloop/data/_data_/_default_/configs/application.ini
```

[From here](https://www.rainloop.net/docs/configuration/)

**Comment by cm-schl** [comment]
Hi,

this should help you: https://github.com/the-djmaze/snappymail/wiki/Admin-Manual#menu-domains
The documentation of Rainloop does not cover the whole functionality of the fork SnappyMail...
